app/templates/DisplayPageLayout.twig

```diff
@@ -56,7 +56,7 @@
     <!-- Header -->
     <header id="header">
         <div class="container clearfix">
-            {$ header | raw $}
+            {% include 'header' %}
```

Comment:
`{$ header $}` is replacing the placeholder with the contents of the context variable `header`.
`{% include 'header' %}` looks for a template named `header`.